contrib/scripts/tinker-mainnet-genesis.sh

```diff
@@ -7,7 +7,7 @@
 CWD="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 
 UMEEMAINNET_GENESIS_PATH="${UMEEMAINNET_GENESIS_PATH:-$CWD/mainnet_tinkered_genesis.json}"
-MAINNET_EXPORTED_GENESIS_URL="${MAINNET_EXPORTED_GENESIS_URL:-"https://storage.googleapis.com/umeedropzone/jul-28-umee-1-export.json.gz"}"
+MAINNET_EXPORTED_GENESIS_URL="${MAINNET_EXPORTED_GENESIS_URL:-"https://storage.googleapis.com/umeedropzone/artemis-mainnet-set-15-exported-genesis.json.gz"}"
```

Comment:
`set-15` or `sep-15`?

Reply:
my bad, in BR it is Setembro 
Going to change that

Reply:
done at https://github.com/umee-network/umee/pull/1392/commits/a21fbd8d0801b029be7f758578b921e8b0dea99e